packages/material-ui-system/src/createBox.js

```diff
@@ -10,7 +10,7 @@ function isEmpty(obj) {
 
 const useTheme = (defaultTheme) => {
   const contextTheme = React.useContext(ThemeContext);
-  return isEmpty(contextTheme) ? defaultTheme : contextTheme;
+  return !contextTheme || isEmpty(contextTheme) ? defaultTheme : contextTheme;
```

Comment:
if `isEmpty` assumes an object then let's call it `isObjectEmpty`. Because I would expect `isEmpty(null)` to be perfectly safe.